.ci/e2e-synthetics.groovy

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'ubuntu-18 && immutable' }
+  environment {
+    REPO = "synthetics"
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    PIPELINE_LOG_LEVEL = 'INFO'
+    SLACK_CHANNEL = '#synthetics-user_experience-uptime'
```

Comment:
Nit: Can be fixed in a follow up but this mixes dashes and underscores

Reply:
IIUC, that particular channel exists, but we don't manage it, do you mean to change the slack channel? Or something in the code itself?

Reply:
Ah. I thought it was a brand-new channel. If it already exists than I said nothing. :)